react-ui/src/components/Common/ColorSchema.scss

```diff
@@ -13,3 +13,5 @@ $emergency-btn-color: rgb(255, 175, 25);
 
 $typical-font-size: 20px;
 $input-font-size: 16px;
+
+$login-signup-button-colour: #b8b1fd; 
```

Comment:
I see you have changed the button colours for the login and signup pages to be consistent with the header colour - as per TA reviews. This looks good - thank you!

Reply:
Thank you for noticing!